core/src/main/java/tc/oc/pgm/teams/TeamModule.java

```diff
@@ -135,6 +135,10 @@ private static TeamFactory parseTeamDefinition(Element el, MapFactory factory)
       throw new InvalidXMLException("Max overfill can not be less then max players.", el);
     }
 
+    if (maxPlayers <= minPlayers) {
+      throw new InvalidXMLException("Min players can not be less (or equal) to max players.", el);
```

Comment:
Min can definetly be equal to max.
If you want a map to play 8v8, with no more and no less players, you set both min and max to 8 in each team